cba_settings_userconfig/cba_settings.sqf

```diff
@@ -739,27 +739,36 @@ js_jc_fa18_showLabels = true;
 force force grad_trenches_functions_allowBigEnvelope = true;
 force force grad_trenches_functions_allowCamouflage = true;
 force force grad_trenches_functions_allowDigging = true;
+force force grad_trenches_functions_allowEffects = true;
 force force grad_trenches_functions_allowGiantEnvelope = true;
+force force grad_trenches_functions_allowHitDecay = true;
 force force grad_trenches_functions_allowLongEnvelope = true;
 force force grad_trenches_functions_allowShortEnvelope = true;
 force force grad_trenches_functions_allowSmallEnvelope = true;
 force force grad_trenches_functions_allowTrenchDecay = false;
 force force grad_trenches_functions_allowVehicleEnvelope = true;
+force force grad_trenches_functions_bigEnvelopeDamageMultiplier = 2;
 force force grad_trenches_functions_bigEnvelopeDigTime = 40;
 force force grad_trenches_functions_bigEnvelopeRemovalTime = -1;
 force force grad_trenches_functions_buildFatigueFactor = 1;
 force force grad_trenches_functions_camouflageRequireEntrenchmentTool = false;
+force force grad_trenches_functions_createTrenchMarker = false;
```

Comment:
```suggestion
force grad_trenches_functions_createTrenchMarker = true;
```
It's a cool feature that should be on by default but disablable

Yes I'm biased as I made this feature

Reply:
For @Shiny-CNTO and @redteddy in case you don't know what that feature does: https://github.com/gruppe-adler/grad_trenches/pull/105

I agree with @Seb105 , should be set to true as it automates tedious marking of player trenches on the map.